ci/scripts/clone-build_ci.sh

```diff
@@ -79,7 +79,7 @@ echo "${commit}" > "../commit"
 # run checkout script
 cd sorc || exit 1
 set +e
-./checkout.sh -c -g -u &>> log.checkout
+./checkout.sh -c -g -u >> log.checkout
```

Comment:
Removing the `&` means stderr is no longer redirected into the log.

Reply:
It was showing an error on `shellcheck`
`log.checkout` and log.build` are not the artifacts one looks at.  The `outfile` in this script contains those.
I'll be happy to add this back.

Reply:
See if shellcheck likes the older form better:
```suggestion
./checkout.sh -c -g -u >> log.checkout 2>&1
```

Reply:
Done